app/controllers/roles_controller.rb

```diff
@@ -29,6 +29,11 @@ class RolesController < CrudController
 
   hide_action :index, :show
 
+  def new
+    @type = @group.default_role
+    super
```

Comment:
Do I have to call super?
Do I need to add arguments? If yes, please correct it and let me know.